### DISCUSSION TITLE
get full articles from a feed that has only a part of the content

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#7024] [Q&A] [answered]
Hi,
[this feed](https://www.oggitreviso.it/feed_prima_pagina_tutte_le_news) provides only part of the article's content, to read it fully on FreshRSS you have to open the original.

[Read You](https://github.com/Ashinch/ReadYou) has an option to analyze the full content, with this enabled it shows correctly the full text and images.

Does FreshRSS provide a similar way to do it?

Thanks

**Comment by Frenzie** [comment]
I don't know what you mean by analyze, but under advanced there are more options.

**Reply by ghost** [reply]
It means that Read You analyse the original article and shows the full content, not just what's in the RSS feed.

I already tried the advanced options, none of them seems to fit this case

**Reply by Frenzie** [reply, marked as answer]
Nothing automated, but most websites can be done with a relatively simple CSS selector.
